lib/mqtt/mqtt.dart

```diff
@@ -0,0 +1,92 @@
+library mqtt;
+
+import 'dart:async';
+
+import 'package:ltbl/util/stream_extensions.dart';
+import 'package:mqtt5_client/mqtt5_client.dart';
+import 'package:mqtt5_client/mqtt5_server_client.dart';
+import 'package:typed_data/typed_buffers.dart';
+
+part './subscription.dart';
+
+typedef ReceivedMessage = MqttReceivedMessage<MqttMessage>;
+
+class MqttStreamClient {
+  late final MqttStreamSubscriptionManager _subscriptionManager;
+  final StreamController<MqttConnectionStatus> _connectionStatusController =
+      StreamController.broadcast();
+  late final Stream<MqttConnectionStatus> connectionStatus;
```

Comment:
This would probably be nicer as a getter
```suggestion
  Stream<MqttConnectionStatus> get connectionStatus => _connectionStatusController.stream;
```